app/models/challenge.rb

```diff
@@ -1,4 +1,4 @@
 class Challenge < ApplicationRecord
-    validates :title, presence: true, format: { with: /[a-zA-Z0-9_-]/}
-    validates :details, presence: true
+    validates :title, presence: true, format: { with: /[a-zA-Z0-9_-]/}, length: { maximum: 50, }
```

Comment:
would also remove , at end, e.g. length: { maximum: 50 }
and everywhere else this occurs; not an issue but not needed either